frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -133,7 +139,7 @@ export const Dashboard = () => {
           } else return 1
         })
       case 'classes-a-z':
-        return [...courseInfo].sort((a, b) => {
+        return courseInfo.sort((a, b) => {
```

Comment:
The `[...courseInfo]` copies the array to avoid mutating the state variable since `.sort()` is in-place in JavaScript. You should probably keep it